releasenotes/notes/44159.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+releaseNotes:
+  - |
+    **Fixed** Fixes selector logic of service and endpoint.
```

Comment:
Can you provide more information about the updates to the selector logic mentioned in the release notes?

Reply:
Done, PTAL :-)